Introduction/Introduction.md

```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
+But nobody like remember the history about the Linux kernel.
+
+It s now installed on millions of systems and used by home users and professionals alike for a wide range of tasks. From miniature embedded systems in wristwatches to massively parallel mainframes, the are countless way of exploiting linux productively.
```

Comment:
- It's
- wrist watches
- there are
- perhaps another word than 'exploiting' is more suitable. For example 'utilizing'?
- Linux